website/blog/2022/04/17/md.md

```diff
@@ -0,0 +1,313 @@
+
+}'
+
+```
+
+We can test our example by simply running the following curl command:
+
+`curl -i http://127.0.0.1:9080/get`
+
+As you can see, there are some additional trace identifiers (like traceId, spanId, parentId) were appended to the headers:
+
+```
+    "X-B3-Parentspanid": "61bd3f4046a800e7",
+
+    "X-B3-Sampled": "1",
+
+    "X-B3-Spanid": "855cd5465957f414",
+
+    "X-B3-Traceid": "e18985df47dab632d62083fd96626692",
+
+```
+
+Then you can use a browser to access `http://127.0.0.1:9411/zipkin`, see traces on the Web UI of Zipkin.
+
+> Note that you need to run the Zipkin instance in order to install Zipkin Web UI. For example, by using docker you can simply run it:
+> `docker run -d -p 9411:9411 openzipkin/zipkin`
+
+[![Zipkin plugin output 1](https://res.cloudinary.com/practicaldev/image/fetch/s---BmZMyXZ--/c_limit%2Cf_auto%2Cfl_progressive%2Cq_auto%2Cw_880/https://dev-to-uploads.s3.amazonaws.com/uploads/articles/av8rdcwwybyfcibbo7xh.png)](https://res.cloudinary.com/practicaldev/image/fetch/s---BmZMyXZ--/c_limit%2Cf_auto%2Cfl_progressive%2Cq_auto%2Cw_880/https://dev-to-uploads.s3.amazonaws.com/uploads/articles/av8rdcwwybyfcibbo7xh.png)
+
+[![Zipkin plugin output 2](https://res.cloudinary.com/practicaldev/image/fetch/s--c9vOR5NU--/c_limit%2Cf_auto%2Cfl_progressive%2Cq_auto%2Cw_880/https://dev-to-uploads.s3.amazonaws.com/uploads/articles/fs6hfbwspk6prcx1xm49.png)](https://res.cloudinary.com/practicaldev/image/fetch/s--c9vOR5NU--/c_limit%2Cf_auto%2Cfl_progressive%2Cq_auto%2Cw_880/https://dev-to-uploads.s3.amazonaws.com/uploads/articles/fs6hfbwspk6prcx1xm49.png)
+
+As you noticed, the recent traces were exposed in the above pictures.
+
+## Summary
+
+As we learned, API Observability is a sort of framework for managing your applications in an API world and Apache APISIX API Gateway plugins can help when observing modern API-driven applications by integrating to several observability platforms. So, you can make your development work focused on core business features instead of building a custom integration for observability tools.
+
+To learn more⤵️
+
+➔ [Download Apache APISIX](https://apisix.apache.org/downloads)
+
+➔ Watch Video Tutorial [Getting Started with Apache APISIX](https://youtu.be/dUOjJkb61so)
+
+➔ Watch Video Tutorial [Getting Started with Apache APISIX Dashboard](https://youtu.be/-9-HZKK2ccI)
+
+➔ Watch Video Tutorial [Overview of Apache APISIX Plugins](https://youtu.be/ixSZA4ILBKQ)
+
+➔ [Install Apache APISIX](https://apisix.apache.org/docs/apisix/how-to-build)
+
+➔ Watch the Video version of the blog post [APIs Observability with Apache APISIX Plugins](https://youtu.be/XK0xcui5BQU)
+
+Community⤵️
+
+🙋 [Join the Apache APISIX Community](https://apisix.apache.org/docs/general/community)
+🐦 [Follow us on Twitter](https://twitter.com/ApacheAPISIX)
+📝 [Find us on Slack](https://join.slack.com/t/the-asf/shared_invite/zt-vlfbf7ch-HkbNHiU_uDlcH_RvaHv9gQ)
+📧 [Mail to us](https://dev@apisix.apache.org/)  
+with your questions
```

Comment:
```suggestion
If you have any questions, feel free to mail [Apache APISIX Community](https://dev@apisix.apache.org/)  
```

Reply:
Because the APISIX Website has details about those(how to enter Slack and GitHub etc, I think this part can detele